### ISSUE TITLE
"unexpected end of JSON input" in case of empty cpu_info parameter of Hypervisor

### ISSUE THREAD

**Issue by ildarsv** [gophercloud/gophercloud#1319]
There is the `"cpu_info": "",` in case of **ironic** hypervisor, as a result I have `unexpected end of JSON input` from the https://github.com/gophercloud/gophercloud/blob/master/openstack/compute/v2/extensions/hypervisors/results.go#L102

```
{
    "hypervisors": [
        {
            "cpu_info": "",
            "current_workload": 0,
            "disk_available_least": 0,
            "free_disk_gb": 0,
            "free_ram_mb": 0,
            "host_ip": "10.3.0.39",
            "hypervisor_hostname": "36a26c3b-f8e1-465d-9fce-c4afbe019b51",
            "hypervisor_type": "ironic",
            "hypervisor_version": 1,
            "id": 150,
            "local_gb": 0,
            "local_gb_used": 0,
            "memory_mb": 0,
            "memory_mb_used": 0,
            "running_vms": 0,
            "service": {
                "disabled_reason": null,
                "host": "bmt01",
                "id": 58
            },
            "state": "up",
            "status": "enabled",
            "vcpus": 0,
            "vcpus_used": 0
        }
    ]
}
```

**Comment by ildarsv** [comment]
Working on it

**Comment by jtopjian** [comment]
Implemented in #1541
